kuma/users/jinja2/users/includes/delete-user-content.html

```diff
@@ -7,7 +7,20 @@
             <p>{{ _('Deleting your account loses any preferences you have set.') }}</p>
             <p>{{ _('Although we can delete your account, any revisions you have made to pages on MDN will remain. Please select how would like us to handle the attribution for your changes.') }}</p>
             {{ form.attributions.errors }}
-            {{ form.attributions }}
+            <ul>
+                {% for field in form.attributions %}
+                    <li>
+                        <label for="{{ field.id_for_label }}">
+                            {{ field.tag() }}
```

Comment:
@callahad btw. this is what tripped me up. In the Django docs[1] it shows this to be `{{ field.tag }}`. When I did that I got `<bound method BoundWidget.tag of <django.forms.boundfield.BoundWidget object at 0x7fb96803b970>>`. I went reading and searching all over the place to try and figure out why the heck I got this output.

In the end the answer was staring me in the face i.e. `bound method BoundWidget.tag`. Method being the keyword here :)

[1] https://docs.djangoproject.com/en/3.0/ref/forms/widgets/#radioselect